2008-10-30  Arnaud Charlet  <charlet@adacore.com>

	* src/cairo-quartz-font.c: Add missing prototype

--- src/cairo-quartz-font.c.old	2008-10-30 11:36:44.000000000 +0100
+++ src/cairo-quartz-font.c	2008-10-30 12:35:26.000000000 +0100
@@ -786,6 +786,8 @@ _cairo_quartz_scaled_font_get_cg_font_re
 cairo_font_face_t *
 cairo_quartz_font_face_create_for_atsu_font_id (ATSUFontID font_id)
 {
+    extern ATSFontRef FMGetATSFontRefFromFont (FMFont iFont);
+
     ATSFontRef atsFont = FMGetATSFontRefFromFont (font_id);
     CGFontRef cgFont = CGFontCreateWithPlatformFont (&atsFont);
     cairo_font_face_t *ff;
